src/database/migrations/20190909032506-create-user.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      firstName: { type: Sequelize.STRING },
```

Comment:
```suggestion
      firstName: { type: Sequelize.STRING, allowNull: false },
```

Reply:
Do this for `lastName`, `email` and `password`

Reply:
Correction received, thanks